.gitignore

```diff
@@ -97,3 +97,13 @@ db.sqlite3
 
 #ds_store
 .ds_store
+
+# Local Files
+.DS_Store
+authors/.DS_Store
+authors/apps/.DS_Store
+authors/apps/authentication/.DS_Store
+authors/apps/authentication/migrations/0001_initial.py
```

Comment:
Remove this line from the `.gitignore` file and Just make sure that you do not stage the changes to the migrations file. Everything else looks okay

Reply:
Thank you for the feedback Joan. Fixed the file.